java/src/main/native/src/TableJni.cpp

```diff
@@ -927,7 +920,7 @@ JNIEXPORT long JNICALL Java_ai_rapids_cudf_Table_writeParquetBufferBegin(
     table_metadata_with_nullability metadata;
     metadata.column_nullable = nullability;
     metadata.column_names = col_names.as_cpp_vector();
-    for (size_t i = 0; i < meta_keys.size(); ++i) {
+    for (auto i = 0; i < meta_keys.size(); ++i) {
```

Comment:
this one is `auto` but others are `int`, could all of them be `auto`?

Reply:
nope, I tried that, it wasn't happy about `auto`